src/app/modules/main/controller.nim

```diff
@@ -229,12 +229,17 @@ proc getActiveSectionId*(self: Controller): string =
   result = self.activeSectionId
 
 proc setActiveSection*(self: Controller, sectionId: string, sectionType: SectionType) =
+  if sectionType == SectionType.Community and
+      not singletonInstance.localAccountSensitiveSettings.getCommunitiesEnabled():
+    # Communities are not supposed to be shown
+    return
+
   self.activeSectionId = sectionId
 
   if(sectionType == SectionType.Chat or sectionType == SectionType.Community):
     # We need to take other actions here, in case of Chat or Community sections like
     # notify status go that unviewed mentions count is updated and so...
-    echo "deal with appropriate service..."
+    discard
```

Comment:
I removed that echo because it was annoying and this `if` is more a TODO than anything

Reply:
@jrainville feel free to remove the entire `if` condition since it does nothing. :)